packages/components/src/components/TextField/TextField.tsx

```diff
@@ -11,55 +11,73 @@ export type TextFieldProps = {
-    return (
-        <View style={[styles.outerContainer, rest.style]} ref={ref}>
-            {label && <Label label={label} />}
-            <View style={styles.innerContainer}>
-                <TextInput
-                    multiline={multiline}
-                    editable={!disabled}
-                    value={contents}
-                    placeholder={placeholder}
-                    onChangeText={(x) => {
-                        setContents(x);
-                        onChange?.(x);
-                    }}
-                    placeholderTextColor={styles.placeholder.color}
-                    style={[multiline ? { minHeight: 60, maxHeight: 60 } : {}, contents ? styles.textInput : styles.placeholder]}
-                ></TextInput>
+export const TextField = React.forwardRef<View, TextFieldProps>(
+    (
+        {
+            label,
+            helperText,
+            onChange,
+            multiline,
+            placeholder,
+            disabled,
+            ...rest
+        },
+        ref
+    ) => {
+        const [contents, setContents] = useState("");
+        const styles = useStyles(themedStyles);
+        return (
+            <View style={[styles.outerContainer, rest.style]} ref={ref}>
+                {label && <Label label={label} />}
+                <View style={styles.innerContainer}>
+                    <TextInput
+                        multiline={multiline}
+                        editable={!disabled}
+                        value={contents}
+                        placeholder={placeholder}
+                        onChangeText={(x) => {
+                            setContents(x);
+                            onChange?.(x);
+                        }}
+                        textAlignVertical="top"
+                        placeholderTextColor={styles.placeholder.color}
+                        style={[
+                            multiline ? { minHeight: 60, maxHeight: 60 } : {},
+                            contents ? styles.textInput : styles.placeholder,
+                        ]}
+                    ></TextInput>
+                </View>
+                {helperText && <Label label={helperText} />}
             </View>
-            {helperText && <Label label={label} />}
-        </View>
-    );
-});
+        );
+    }
+);
 
 const themedStyles = EDSStyleSheet.create((theme) => {
     return {
         innerContainer: {
             backgroundColor: theme.colors.container.default,
             padding: 12,
-
             borderBottomWidth: theme.geometry.border.borderWidth,
             borderBottomColor: theme.colors.border.medium,
             marginTop: 4,
-            marginBottom: 4,
+            marginBottom: 8,
             marginLeft: -12,
         },
         outerContainer: {
             marginLeft: 12,
         },
         textInput: {
             color: theme.colors.text.primary,
+            fontFamily: theme.typography.basic.p.fontFamily,
```

Comment:
You can do

```ts
textInput: {
    color: theme.colors.text.primary,
    ...theme.typography.basic.p
}
```

to destructure everything in the font token. Makes the style a little bit more dynamic